example/gardener-local/gardenlet/values-kind-ha-multi-zone.yaml

```diff
@@ -38,3 +38,6 @@ config:
         domain: ingress.local-ha-multi-zone.seed.local.gardener.cloud
       secretRef:
         name: seed-local-ha-multi-zone
+      settings:
+        topologyAwareRouting:
+          enabled: true
```

Comment:
Should we also add it to `example/provider-local/seed-kind-ha-multi-zone/local/seed.yaml`?